ros/api/v1/recommendations.py

```diff
@@ -53,7 +54,25 @@ def get(self, host_id):
                 if rule_data:
                     rule_dict = rule_data.__dict__
                     recommendation = {}
+                    instance_price = 0
```

Comment:
```suggestion
                    instance_price = ''
```

Reply:
String won't work here `instance_price += rule_hit_details.get('price')`. we're using shorthand operator for addition